web/auth/jwt_test.go

```diff
@@ -1,6 +1,67 @@
 package auth_test
 
-import "testing"
+import (
+	"testing"
 
-func TestUpdateJWT(t *testing.T) {
+	"github.com/autograde/quickfeed/ag"
+	pb "github.com/autograde/quickfeed/ag"
+	"github.com/autograde/quickfeed/web/auth"
+)
+
+func TestUpdateJWTClaims(t *testing.T) {
+	tokens := []*pb.UpdateTokenRecord{
+		{
+			ID:     1,
+			UserID: 1,
+		},
+		{
+			ID:     2,
+			UserID: 2,
+		},
+		{
+			ID:     3,
+			UserID: 3,
+		},
+	}
+	manager := &auth.TokenManager{TokensToUpdate: tokens}
+
+	claimsToUpdate := auth.Claims{
+		UserID:  2,
+		Admin:   false,
+		Courses: map[uint64]ag.Enrollment_UserStatus{1: pb.Enrollment_STUDENT},
+	}
+
+	claimsNoUpdate := auth.Claims{
+		UserID:  10,
+		Admin:   true,
+		Courses: make(map[uint64]pb.Enrollment_UserStatus, 0),
+	}
+
+	idNoUpdate := manager.JWTUpdateRequired(&claimsNoUpdate)
+	if idNoUpdate != -1 {
+		t.Errorf("expected index -1 (update not required), got %d", idNoUpdate)
+	}
+	idToUpdate := manager.JWTUpdateRequired(&claimsToUpdate)
+	if idToUpdate != int(claimsToUpdate.UserID) {
+		t.Errorf("expected user ID %d, got %d", claimsToUpdate.UserID, idToUpdate)
```

Comment:
Please adopt the standard approach for [reporting test failures](https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures).